Features/WatchList/WatchlistViewController.swift

```diff
@@ -68,8 +78,7 @@ extension WatchlistViewController: WatchlistButtonDelegate {
         guard let indexPath = contentView.tableView.indexPath(for: cell) else { return }
         
         if let movie = movies[safe: indexPath.row] {
```

Comment:
This is not from this PR but I just noticed, `movie` is a prerequisite for the method to run so it should be part of the guard statement at the beginning